src/components/views/ViewDataTable/cells/PersonNotesViewCell.tsx

```diff
@@ -0,0 +1,37 @@
+import { FunctionComponent } from 'react';
+import { Typography } from '@material-ui/core';
+
+import { ViewGridCellParams } from '.';
+
+
+export type PersonNotesViewCellParams = ViewGridCellParams<{
+    created: string;
+    id: number;
+    text: string;
+}[] | null>;
```

Comment:
Why have the params include the possible null type? I would put `params` as an optional prop? Maybe this will make sense to me elsewhere, but is `null` a valid value to be considered "params"